src/test/java/io/confluent/connect/jdbc/dialect/BaseDialectTest.java

```diff
@@ -313,11 +314,16 @@ public void bindFieldPrimitiveValues() throws SQLException {
     verifyBindField(++index, Schema.BYTES_SCHEMA, new byte[]{42}).setBytes(index, new byte[]{42});
     verifyBindField(++index, Schema.BYTES_SCHEMA, ByteBuffer.wrap(new byte[]{42})).setBytes(index, new byte[]{42});
     verifyBindField(++index, Schema.STRING_SCHEMA, "yep").setString(index, "yep");
-    verifyBindField(++index, Decimal.schema(0), new BigDecimal("1.5").setScale(0, BigDecimal.ROUND_HALF_EVEN)).setBigDecimal(index, new BigDecimal(2));
-    verifyBindField(++index, Date.SCHEMA, new java.util.Date(0)).setDate(index, new java.sql.Date
-        (0), DateTimeUtils.UTC_CALENDAR.get());
-    verifyBindField(++index, Time.SCHEMA, new java.util.Date(1000)).setTime(index, new java.sql.Time(1000), DateTimeUtils.UTC_CALENDAR.get());
-    verifyBindField(++index, Timestamp.SCHEMA, new java.util.Date(100)).setTimestamp(index, new java.sql.Timestamp(100), DateTimeUtils.UTC_CALENDAR.get());
+    verifyBindField(++index, Decimal.schema(0),
+            new BigDecimal("1.5").setScale(0, BigDecimal.ROUND_HALF_EVEN))
+            .setBigDecimal(index, new BigDecimal(2));
```

Comment:
Our style is a bit strange to those not familiar with it, but these lines would be formatted:
```
    verifyBindField(
      ++index, 
      Decimal.schema(0),
      new BigDecimal("1.5").setScale(0, BigDecimal.ROUND_HALF_EVEN)
    ).setBigDecimal(index, new BigDecimal(2));
```

Reply:
:+1: